src/event.js

```diff
@@ -504,6 +565,77 @@ jQuery.event = {
 	}
 };
 
+// Ensure the presence of an event listener that handles manually-triggered
+// synthetic events by interrupting progress until reinvoked in response to
+// *native* events that it fires directly, ensuring that state changes have
+// already occurred before other listeners are invoked.
+function leverageNative( el, type, forceAdd, allowAsync ) {
```

Comment:
This will hurt readability but since `forceAdd` may be only `false` or `returnTrue`, we could squash it with `allowAsync`, changing the signatures to:
```js
leverageNative( el, type, allowAsync ); // old `leverageNative( el, type, false, allowAsync )`
leverageNative( el, type, false ); // old `leverageNative( el, type, returnTrue )`
```
That may save some size.

Reply:
Yes, I think I meant something like that. If this doesn't help size then consider the request moot.